ci-operator/step-registry/azure/provision/disk-encryption-set/azure-provision-disk-encryption-set-commands.sh

```diff
@@ -105,10 +105,12 @@ fi
 cluster_sp_id=$(cat "${AZURE_AUTH_LOCATION}" | jq -r ".clientId")
 role_name="Owner"
 if [[ "${ENABLE_MIN_PERMISSION_FOR_DES}" == "true" ]]; then
-    role_name=$(< "${SHARED_DIR}/azure_custom_role_name")
-    if [ -f "${SHARED_DIR}/azure_sp_id" ]; then
```

Comment:
Seem like `azure_sp_id` also the same case as `azure_custom_role_name`, it might have multiple lines.

`azure-provision-vnet` and `ipi-conf-azure-resourcegroup` is reading its contents.

Reply:
good catch, updated to read file `azure_minimal_permission`